orchestra/json_schemas/todos.py

```diff
@@ -0,0 +1,11 @@
+import jsl
+
+
+class TodoSchema(jsl.Document):
+    id = jsl.IntField(required=True)
+    description = jsl.StringField(required=True)
+    items = jsl.DocumentField('TodoSchema', required=True)
```

Comment:
the children won't be sorted if you do this, right?

you might end up needing an ArrayList of (slug, Todo) pairs?